lib/robots/viomi/capabilities/ViomiZoneCleaningCapability.js

```diff
@@ -0,0 +1,58 @@
+const ZoneCleaningCapability = require("../../../core/capabilities/ZoneCleaningCapability");
```

Comment:
This file is already part of #675 IIRC, can you remove it from this PR?

Reply:
I think rebasing would get rid of this change from this MR. No need to manually delete those files, and here it would actually create conflicts.

Reply:
yes, rebase should clean this